### ISSUE TITLE
Kubernetes v1.15.0-beta.0 generation fails

### ISSUE THREAD

**Issue by teyckmans** [kuberig-io/kuberig-dsl-kubernetes#1]
[build-output.log](https://github.com/teyckmans/kuberig-dsl-kubernetes/files/3199144/build-output.log)

[JSONSchemaProps-invalid.txt](https://github.com/teyckmans/kuberig-dsl-kubernetes/files/3199156/JSONSchemaProps-invalid.txt)

Part of the generated JSONSchemaProps.kt file that causes a compilation failure.
```kotlin

    val x-kubernetes-embedded-resource : Boolean?,

    val x-kubernetes-int-or-string : Boolean?,

    val x-kubernetes-preserve-unknown-fields : Boolean?)
```

**Comment by teyckmans** [comment]
We need an upstream fix for this in kuberig-dsl

**Comment by teyckmans** [comment]
Upstream issue created https://github.com/teyckmans/kuberig-dsl-kubernetes/issues/1

**Comment by teyckmans** [comment]
upstream issue is solved and releases with v0.0.11.
